src/app/screens/Keysend/index.tsx

```diff
@@ -46,15 +47,6 @@ function Keysend() {
       : +amountSat > (auth?.account?.balance || 0);
   const rangeExceeded = +amountSat < amountMin;
 
-  useEffect(() => {
```

Comment:
after passing value for such sats presets and converting them. if dualCurrencyField returns fiatValue then we don't need this useEffect. else we need to explicitly convert the value via such useEffect so that right fiatValue is shown in successCard